website/docs/Capabilities.md

```diff
@@ -61,6 +61,24 @@ If the driver `binary` is set, WebdriverIO won't attempt to download a driver bu
 
 :::
 
+#### `wdio:maxInstances`
```

Comment:
I think this is a bit misleading as these caps are not related to driver options, maybe the section could look like this:

```md
### WebdriverIO Capabilities to manage browser driver options

WebdriverIO defines a set of own capabilities to manage its automated session:

#### `wdio:chromedriverOptions`

Specific options passed into Chromedriver when starting it.

#### `wdio:geckodriverOptions`

Specific options passed into Geckodriver when starting it.

#### `wdio:edgedriverOptions`

Specific options passed into Edgedriver when starting it.

#### `wdio:safaridriverOptions`

Specific options passed into Safari when starting it.

#### `wdio:maxInstances`

Maximum number of total parallel running workers for the specific browser/capability. Takes precedence over [maxInstances](#configuration#maxInstances) and [maxInstancesPerCapability](configuration/#maxinstancespercapability).

Type: `number`

#### `wdio:specs`

Define specs for test execution for that browser/capability. Same as the [regular `specs` configuration option](configuration#specs), but specific to the browser/capability. Takes precedence over `specs`.

Type: `(String | String[])[]`

#### `wdio:exclude`

Exclude specs from test execution for that browser/capability. Same as the [regular `exclude` configuration option](configuration#exclude), but specific to the browser/capability. Takes precedence over `exclude`.

Type: `String[]`

WebdriverIO manages installing and running browser driver for you. WebdriverIO uses a custom capability that allows you to pass in parameters to the driver.

#### Common Driver Options
...
```

Reply:
kk, moved and expanded the section as you suggested.